fms/models/llama.py

```diff
@@ -542,11 +550,11 @@ def _hf_sd_to_fms_sd(hf_sd: Mapping) -> Mapping:
                 head_size = 128  # every other Llama model in existence
             nheads = int(temp.size(0) / head_size)
 
-            temp = (
-                temp.view(nheads, 2, -1, temp.size(1))
-                .transpose(1, 2)
-                .reshape(*temp.size())
-            )
+            if temp.dim() == 2:
```

Comment:
might help to have a comment explaining the cases where these different reshapes are happening and why